application/api/dos/models.py

```diff
@@ -0,0 +1,147 @@
+from django.db import models
+
+
+class Services(models.Model):
+
+    uid = models.CharField(max_length=255)
+    name = models.CharField(max_length=255)
+    # TODO Not used attributes of the services table, will be removed once model stability is confirmed
```

Comment:
I think we are stable enough now, lets remove these commented lines from the code.

Reply:
Agree, will remove commented out lines